src/views/partials/dashboard.js

```diff
@@ -0,0 +1,3 @@
+export default data => `
+<h1>This is the breach dashboard page</h1>
```

Comment:
The dashboard title is hidden behind the Logo / Sign Up header for me. Should the body have some padding at the top to make room for the header?

Reply:
Ah yes, this was a quick POC by Joe that will be updated in a future PR 👍